test/testModel.cc

```diff
@@ -1357,16 +1377,18 @@ SUITE(GodleyTableWindow)
       CHECK_EQUAL(1,insertIdx);
       keyPress(XK_Left,"");
       CHECK_EQUAL(0,insertIdx);
-      keyPress(XK_Left,"");
-      CHECK_EQUAL(0,selectedCol);
+      // (1,0) cell no longer selectable
+      //keyPress(XK_Left,"");
+      //CHECK_EQUAL(0,selectedCol);    
       keyPress(XK_Tab,"");
-      CHECK_EQUAL(1,selectedCol);
+      CHECK_EQUAL(2,selectedCol);  // selectedCol is still equal to 1 since above operation attempting to access cell (1,0) has not been performed
       keyPress(XK_Tab,"");
-      CHECK_EQUAL(2,selectedCol);
-      keyPress(XK_Right,"");
       CHECK_EQUAL(3,selectedCol);
-      keyPress(XK_Right,"");
+      keyPress(XK_Right,"");     // check wrap around
       CHECK_EQUAL(0,selectedCol);
+      // (1,0) cell no longer selectable
+      //keyPress(XK_Right,"");
+      //CHECK_EQUAL(0,selectedCol);
```

Comment:
I don't understand what this code block is doing. Probably just dead code that should be removed.

Reply:
I've removed it. I left it there to show how the test has been modified, but you're right that it is now redundant.